src/yoroi-wallets/utils.test.ts

```diff
@@ -214,3 +220,53 @@ describe('Entries', () => {
     } as YoroiAmounts)
   })
 })
+
+describe('Utxos', () => {
+  it('toAmounts', () => {
```

Comment:
scenarios:

- empty utxos should return the primarty token with 0
- utxos without assets, should sum up primary token 
- utxos with assets, should sum up primary token + assets by id

Reply:
Done. Thanks!